clippy_config/src/conf.rs

```diff
@@ -613,6 +613,10 @@ define_Conf! {
     /// - Use `".."` as part of the list to indicate that the configured values should be appended to the
     /// default configuration of Clippy. By default, any configuration will replace the default value
     (allowed_prefixes: Vec<String> = DEFAULT_ALLOWED_PREFIXES.iter().map(ToString::to_string).collect()),
+    /// Lint: UNNECESSARY_MIN_OR_MAX.
+    ///
+    /// Whether to also run the lints on external defined consts.
+    (allowed_external_crates: bool = false),
```

Comment:
I think it would be fine to have no config, it seems like it would be very rare that an external crate would redefine a constant to be MIN/MAX in a way that the user wants to trigger the lint